modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/css/_admin.scss

```diff
@@ -226,4 +226,14 @@ $toolbarZIndex: 971;
 	.lfr-search-container-wrapper .list-group .list-group-item h2.h5 {
 		font-weight: 600;
 	}
+
+	.management-bar {
+		.nav-item {
+			padding: 0 0.25rem;
+
+			&:last-child {
+				padding-right: 0;
+			}
+		}
+	}
```

Comment:
We can use the gap, [column-gap](https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap) in this case to separate the `.nav-item`.

```
	.management-bar {
		.nav-item {
			column-gap: 0.25rem;
		}
	}
```

And avoid nesting each selector 



```
	.management-bar .nav-item {
		column-gap: 0.25rem;
	}
```

Reply:
I didn't know about` column-gap,` but it is not working for me... maybe because of the display, or because these are `<li>` tags...¿?. For now I've removed the nesting level.

Reply:
Sorry my fault, the `column-gap` attribute work on the parent, not on children. 

```
	.management-bar {
		column-gap: 0.25rem;
	}
	
```

Sorry I missed the comment yesterday 😢